src/node_contextify.cc

```diff
@@ -740,11 +740,10 @@ void ContextifyScript::New(const FunctionCallbackInfo<Value>& args) {
   if (*TRACE_EVENT_API_GET_CATEGORY_GROUP_ENABLED(
           TRACING_CATEGORY_NODE2(vm, script)) != 0) {
     Utf8Value fn(isolate, filename);
-    TRACE_EVENT_NESTABLE_ASYNC_BEGIN1(
-        TRACING_CATEGORY_NODE2(vm, script),
-        "ContextifyScript::New",
-        contextify_script,
-        "filename", TRACE_STR_COPY(*fn));
+    TRACE_EVENT_BEGIN1(TRACING_CATEGORY_NODE2(vm, script),
```

Comment:
I was wondering how costly it is to construct the Utf8Value object in the line above. If it's in the range of microseconds or less, we could remove the if conditional and turn this into a `TRACE_EVENT0` too and make use of RAII instead of calling `TRACE_EVENT_END0` before all the return statements.

Reply:
Basically, Utf8Value is going to copy the string content and the filename is a user input so I don't think it is a good idea to do it in all cases just for tracing.